### ISSUE TITLE
Ensure exports match glob

### ISSUE THREAD

**Issue by bennypowers** [bluwy/publint#27]
Cool package! thanks for publishing!

Given this dir structure in a package root, I want to ensure that `exports` includes all of those files.

```
elements
├── pf-accordion
│   ├── pf-accordion-header.js
│   ├── pf-accordion.js
│   └── pf-accordion-panel.js
├── pf-avatar
│   └── pf-avatar.js
├── pf-badge
│   └── pf-badge.js
└── pf-tooltip
    └── pf-tooltip.js

20 directories, 26 files
```

So linting with an option like `exportGlobs: ['elements/*/pf-*.js']`,

This `"exports"` field should fail
```js
{ '.': './pfe.js' }
```
and this one should pass:
```js
{
  '.': './pfe.js',
  './pf-accordion/pf-accordion-header.js': './pf-accordion/pf-accordion-header.js',
  './pf-accordion/pf-accordion-panel.js': './pf-accordion/pf-accordion-panel.js',
  './pf-accordion/pf-accordion.js': './pf-accordion/pf-accordion.js',
  './pf-avatar/pf-avatar.js': './pf-avatar/pf-avatar.js',
  './pf-badge/pf-badge.js': './pf-badge/pf-badge.js',
  './pf-tooltip/pf-tooltip.js': './pf-tooltip/pf-tooltip.js'
}
```

**Comment by bluwy** [comment]
Hi, I'm not really sure what you're requesting here, how does 

> So linting with an option like `exportGlobs: ['elements/*/pf-*.js']`,

work? I did notice though that I didn't handle glob subpath patterns with multiple `*` (only supports one `*` now), maybe you're referring to this issue?
